gwvolman/tasks.py

```diff
@@ -166,8 +166,15 @@ def remove_volume(payload):
 
 
 @app.task
-def build_image(imageId, imageTag, sourceUrl):
-    """Build docker image from WT Image object and push to a registry."""
+def build_image(imageId, imageTag, sourceUrl, fileId):
```

Comment:
If you already check whether it's `None` or not, could you make it kwarg with `None` as default?